models/perceptron.py

```diff
@@ -30,10 +27,10 @@ def parameters(self):
     def get_model_output(self, X):
         preactivation = T.dot(X, self.W) + self.b
         probs = T.nnet.softmax(preactivation)
-        return probs
+        return probs, OrderedDict()
 
     def use(self, X):
-        probs = self.get_model_output(X)
+        probs = self.get_model_output(X)[0]
```

Comment:
You can make it explicit that we don't care about the second value returned:
`probs, _ = self.get_model_output(X)`